### ISSUE TITLE
Villagers merging

### ISSUE THREAD

**Issue by ImArkdia** [Nathat23/StackMob-3#271]
My players frustrate when theyr villagers merge.

I just want a lane in the config about villagers or help with this.

**Comment by jstnf** [comment]
There's a setting in the config where you can blacklist certain mobs stacking. Have you tried blacklisting Villagers?

```

....

# Don't stack if the entity is one of these types.
# List of the bukkit names can be found at https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/EntityType.html
no-stack-types:
  - VILLAGER

....

```

**Comment by ImArkdia** [comment]
Yes i tried,tried even to use the exact villager profession on blacklist,but it stills merge them.

**Comment by jstnf** [comment]
Don't know if it'll help you out, but I've whitelisted Pigmen to only stack on my server and it seems to work correctly. Perhaps you could try whitelisting every mob *except* Villagers, and see if that fixes it?
